capabilities/promtail/values/values.yaml

```diff
@@ -0,0 +1,166 @@
+config:
+  clients:
+    - url: 'http://loki.loki.svc.cluster.local:3100/loki/api/v1/push'
+# config:
+#   clients:
+#     {{- if eq .Values.loki.strategy "monolith" }}
+#     - url: http://logging-loki.logging.svc.cluster.local:3100/loki/api/v1/push
+#     {{- else }}
+#     - url: http://logging-loki-write.logging.svc.cluster.local:3100/loki/api/v1/push
+#     {{- end }}
+
+  snippets:
+    scrapeConfigs: >
+      - job_name: kubernetes-pods
```

Comment:
I think we can nuke most of this except for the node level stuff (journal/systemd), and instead add those pieces via `snippets.extraScrapeConfigs`.